.dockerignore

```diff
@@ -0,0 +1,14 @@
+.bintray.json
+.dockerignore
+.git
+.gitignore
+.hound.yml
+.jshintrc
+.travis.yml
+dhcpd.conf
```

Comment:
No, it doesn't use that version of dhcp.conf. That is provided here: https://github.com/RackHD/RackHD/pull/86/files#diff-007d36abd8a3deadeee5749bf1f84c98

I'm trying to limit the docker containers to the the bare essential files. When you `ls` the directory in the container you should only get:
```
usxxgardij1m1:docker poultr2$ docker run -it rackhd/on-dhcp-proxy /bin/sh
/RackHD/on-dhcp-proxy # ls
debian        index.js      lib           node_modules  package.json
/RackHD/on-dhcp-proxy # 
```